src/runtime/hexagon/rpc/hexagon/rpc_server.cc

```diff
@@ -211,6 +216,7 @@ const tvm::runtime::PackedFunc get_runtime_func(const std::string& name) {
 void reset_device_api() {
   const tvm::runtime::PackedFunc api = get_runtime_func("device_api.hexagon.v2");
   tvm::runtime::Registry::Register("device_api.hexagon", true).set_body(api);
+  tvm::runtime::Registry::Register("device_api.cpu", true).set_body(api);
```

Comment:
Just curious, why this is necessary? A comment would be nice.

Reply:
done.